app/main.py

```diff
@@ -2,5 +2,25 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    caches = {}
+
+    def inner(*args) -> None:
+        if args in caches:
```

Comment:
add `functools.wraps`

Also, what if functions take key arguments?

Reply:
@vsmutok what do u mean? i don’t get it